lib/MusicBrainz/Server/Report/NotVariousArtists.pm

```diff
@@ -0,0 +1,47 @@
+package MusicBrainz::Server::Report::NotVariousArtists;
+use Moose;
+use MusicBrainz::Server::Constants qw( $VARTIST_ID );
+
+with 'MusicBrainz::Server::Report::RecordingReport',
+     'MusicBrainz::Server::Report::FilterForEditor::RecordingID';
```

Comment:
Looking at lib/MusicBrainz/Server/Report/FilterForEditor/RecordingID.pm it should filter them by artists you're subscribed to.